tests/string_from.cpp

```diff
@@ -15,6 +15,8 @@ bool same(const L& l, const R& r) {
 
 using namespace antler::string;
 
+TEST_CASE("Testing antler::string")
```

Comment:
unused test for deleted code. please remove

Reply:
Fixed in ab2fd78